content/en/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes.md

```diff
@@ -21,9 +21,12 @@ as for readiness probes, but with a higher failureThreshold. This ensures that t
 is observed as not-ready for some period of time before it is hard killed.
 
 The kubelet uses readiness probes to know when a container is ready to start
-accepting traffic. A Pod is considered ready when all of its containers are ready.
-One use of this signal is to control which Pods are used as backends for Services.
-When a Pod is not ready, it is removed from Service load balancers.
+accepting traffic. One use of this signal is to control which Pods are used as
+backends for Services. A Pod is considered ready when its `Ready` [condition](/docs/concepts/workloads/pods/pod-lifecycle/#pod-conditions)
+is true. When a Pod is not ready, it is removed from Service load balancers.
+A Pod's `Ready` condition is false when the Pod's Node is `NotReady`, when one
```

Comment:
Is `NotReady` a condition of Node objects?

Reply:
good catch, I was trying to cut a corner here by reporting the short status output. However, there's no `NotReady` condition, only ` Ready` one:

https://github.com/kubernetes/kubernetes/blob/b6b7abc871a55ce26bc62c0d5452b73077364395/pkg/controller/nodelifecycle/node_lifecycle_controller.go#L744
That comes from https://github.com/kubernetes/kubernetes/blob/b6b7abc871a55ce26bc62c0d5452b73077364395/pkg/controller/nodelifecycle/node_lifecycle_controller.go#L832 and it looks at the `Ready` condition:
https://github.com/kubernetes/kubernetes/blob/b6b7abc871a55ce26bc62c0d5452b73077364395/pkg/controller/nodelifecycle/node_lifecycle_controller.go#L840

I'm going to change it in my new suggestion